### DISCUSSION TITLE
Dependent Queries

### DISCUSSION THREAD

**Discussion by mightym** [TanStack/query#1542] [Q&A] [answered]
Hey everyone, I have a quick question. Not sure if I'm doing something wrong on dependent queries. This is my code:

```
  const userQuery = useQuery('users', getMe);
  const postQuery = useQuery(
    ['posts', userQuery.data?.postID],
    () => getPostDetail(userQuery.data.postID), {
      enabled: userQuery.data?.postID
    }
  );
  ```
It works well but due to the fact that initially `userQuery.data.postID` is null, I'm seeing this in my queries:

<img width="201" alt="1" src="https://user-images.githubusercontent.com/11863/103287846-d2ebba80-49e3-11eb-9030-9e072dcc2dbf.png">

Is this ok/expected or am I doing something wrong? 

Thanks in advance!

**Comment by tannerlinsley** [comment, marked as answer]
This is totally fine :)

**Reply by TkDodo** [reply]
Maybe we could / should visualise disabled queries differently in the devtools?

**Reply by mightym** [reply]
@tannerlinsley great thanks. I feel relieved :)
